assets/stylesheets/screens/_dashboard.scss

```diff
@@ -14,4 +24,16 @@ body.dashboard.index {
       @apply w-full;
     }
   }
+
+  #upload-file-button {
```

Comment:
Why using an ID here instead of a class? Use an ID for the aria-control but then use a CSS class name for styling. 

Also ID should be written in `camelCase` (refer to [Compass](https://nimblehq.co/compass/development/code-conventions/html/#ids-and-class-names))

Reply:
I've changed this to class name already